static/src/javascripts/projects/common/modules/crosswords/confirm-button.js

```diff
@@ -1,20 +1,27 @@
 define([
     'react',
-    'classnames',
     'lodash/objects/assign'
 ], function (
     React,
-    classNames,
     assign
 ) {
     var ConfirmButton = React.createClass({
+
         getInitialState: function () {
             this.timeout = this.props.timeout || 2000;
             return {
                 confirming: false
             };
         },
 
+        classNames : function(props) {
```

Comment:
It does feel a bit odd to have to inline these, could we put the `className` helper in `crossword` module path to see if we can fix the test compilation issues ?